mlflow/pyfunc/__init__.py

```diff
@@ -2224,7 +2224,9 @@ def save_model(
         python_model:
             An instance of a subclass of :class:`~PythonModel` or a callable object with a single
             argument (see the examples below). The passed-in object is serialized using the
-            CloudPickle library. Any dependencies of the class should be included in one of the
+            CloudPickle library. The python_model can also be a file path to the PythonModel
+            which logs the model as code artifact rather than serializing the model object.
```

Comment:
```suggestion
            which defines the model as code artifact rather than serializing the model object.
```
nit: just for accuracy - log_model call is not in the model file. (same for the log_model docstring)